<ui version="4.0" >
 <class>CollectionGeneralPropertiesPage</class>
 <widget class="QWidget" name="CollectionGeneralPropertiesPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Name:</string>
     </property>
     <property name="buddy" >
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="KLineEdit" name="nameEdit" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="customIconCheckbox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>&amp;Use custom icon:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="KIconButton" name="customIcon" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="icon" >
      <string>folder</string>
     </property>
     <property name="iconSize" >
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QGroupBox" name="statsBox" >
     <property name="title" >
      <string>Statistics</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Content:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="countLabel" >
        <property name="text" >
         <string>0 objects</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="sizeLabel" >
        <property name="text" >
         <string>0 Byte</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customIconCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>customIcon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>245</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
